### ISSUE TITLE
MaxwellNet: Physics-driven deep neural network training based on Maxwell's equations

### ISSUE THREAD

**Issue by DaigoOUE** [DaigoOUE/arXiv-check#26]
Joowon Lim, Demetri Psaltis

> Maxwell's equations govern light propagation and its interaction with matter. Therefore, the solution of Maxwell's equations using computational electromagnetic simulations plays a critical role in understanding light-matter interaction and designing optical elements. Such simulations are often time-consuming and recent activities have been described to replace or supplement them with trained deep neural networks (DNNs). Such DNNs typically require extensive, computationally demanding simulations using conventional electromagnetic solvers to compose the training dataset. In this paper, we present a novel scheme to train a DNN that solves Maxwell's equations speedily and accurately without relying on other computational electromagnetic solvers. Our approach is to train a DNN using the residual of Maxwell's equations as the physics-driven loss function for a network that finds the electric field given the spatial distribution of the material property. We demonstrate it by training a single network that simultaneously finds multiple solutions of various aspheric micro-lenses. Furthermore, we exploit the speed of this network in a novel inverse design scheme to design a micro-lens that maximizes a desired merit function. We believe that our approach opens up a novel way for light simulation and optical design of photonics devices.

https://arxiv.org/abs/2107.06164

**Comment by DaigoOUE** [comment]
maxwellを解くディープニューラルネット🧠

**Comment by DaigoOUE** [comment]
FDTDとかFEMが要らないや！という時代が来るのかもしれない…
